doc/source/developing/testing.rst

```diff
@@ -3,503 +3,400 @@
+
+When the answer tests are run, their output is compared to the gold standard answers to ensure that the results of operating on data do not change over time.
 
 .. _run_answer_testing:
 
 How to Run the Answer Tests
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The very first step is to make a directory and copy over the data against which
-you want to test. Next, add the config parameter ``test_data_dir`` pointing to
-directory with the test data you want to test with, e.g.:
+In order to run the answer tests locally:
+
+* Create a directory to hold the data required by the answer tests one wishes to run
+
+* Fill this directory with the required data
+
+Next, yt needs to be made aware of where it can find the data. This is done by setting the config parameter ``test_data_dir`` to the
+directory with the test data downloaded from https://yt-project.org/data/. For example,
 
 .. code-block:: bash
 
    $ yt config set yt test_data_dir /Users/tomservo/src/yt-data
 
-We use a number of real-world datasets for the tests that must be downloaded and
-unzipped in the ``test_data_dir`` path you have set. The test datasets, can be
-downloaded from https://yt-project.org/data/. We do not explicitly list the
-datasets we use in the tests here because the list of necessary datasets changes
-regularly, instead you should take a look at the tests you would like to run and
-make sure that the necessary data files are downloaded before running the tests.
+To run the answer tests locally, you must first generate a set of gold standard answer files using a "known good" version of yt. Once done, then update to the version of yt you want to test and
+run the tests again so that the results can be compared to those contained in the gold standard answer files.
 
-To run the answer tests, you must first generate a set of test answers locally
-on a "known good" revision, then update to the revision you want to test, and
-run the tests again using the locally stored answers.
+As an example, let's focus on running the answer tests for the tipsy frontend.
 
-Let's focus on running the answer tests for a single frontend. It's possible to
-run the answer tests for **all** the frontends, but due to the large number of
-test datasets we currently use this is not normally done except on the yt
-project's contiguous integration server.
+.. note::
+    It's possible to run the answer tests for **all** the frontends, but due to the large number of test datasets we currently use this is not normally done except on the yt project's contiguous integration server.
 
 .. code-block:: bash
 
    $ cd $YT_GIT
-   $ nosetests --with-answer-testing --local --local-dir $HOME/Documents/test --answer-store --answer-name=local-tipsy yt.frontends.tipsy
+   $ pytest --with-answer-testing --answer-store --local-dir="$HOME/Documents/test" -k "TestTipsy"
```

Comment:
where are they running this command from? The top yt git directory? Do they need to specify the path to the Tipsy tests?